index.php

```diff
@@ -8,5 +8,29 @@
 echo esc_html( get_template_part( 'template-parts/pagination', 'infinite-scroll' ) );
 ?>
 
+<section class="loginpage-background">
```

Comment:
1. Niepoprawny BEM, jeśli zaczynasz nazwę klasu od "loginpage", wtedy poprawnie będzie "loginpage__background", ""loginpage__form-description" i td, jeśli od  "loginpage-form", to "loginpage-form__background" i "loginpage-form__description"
2. Niepoprawnie użyty label do input